pkgs/by-name/sw/switch-to-configuration-ng/src/Cargo.toml

```diff
@@ -12,7 +12,9 @@ glob = "0.3.1"
 log = "0.4.21"
 nix = { version = "0.28.0", features = ["fs", "signal"] }
 regex = "1.10.4"
-rust-ini = "0.21.0"
+rust-ini = { git = "https://github.com/jmbaur/rust-ini", branch = "multi-line", features = [
+  "inline-comment",
+] }
```

Comment:
That's the goal. I plan on PR'ing it soon. The "inline-comment" feature is actually a feature unrelated to my work, but we need it anyways.

Reply:
Sounds good. Note that due to how Cargo unifies features across the dependency tree it’s somewhat dangerous to use them to change behaviour like this, but I guess that’s a problem with the flags this crate has already.

Reply:
So I have a PR for upstream [here](https://github.com/zonyitoo/rust-ini/pull/137). @emilazy Assuming I have changes to make to that PR to be acceptable for merging upstream, do you think it would be best to pin to a different branch here with the same changes? I think github keeps unreachable git objects on their end, but might be best to not take that chance.

Reply:
Upstream PR is merged! Updated cargo dependency to point to that rev